File: code-samples/messaging/global-business/cancel-message.java
import com.messente.ApiClient;
import com.messente.ApiException;
import com.messente.api.*;
import com.messente.auth.HttpBasicAuth;

public class CancelMessage {
    public static void main(String[] args) {
        ApiClient apiClient = new ApiClient();
        OmnimessageApi apiInstance = new OmnimessageApi(apiClient);

        HttpBasicAuth basicAuth = (HttpBasicAuth) apiClient.getAuthentication("basicAuth");
        basicAuth.setUsername("YOUR_MESSENTE_API_USERNAME");
        basicAuth.setPassword("YOUR_MESSENTE_API_PASSWORD");

        try {
            apiInstance.cancelScheduledMessage("<omnimessage_id>");
            System.out.println("Scheduled omnimessage cancelled");
        } catch (ApiException e) {
            System.err.println("Exception when cancelling an omnimessage");
            System.err.println(e.getResponseBody());
        }
    }
}

File: code-samples/messaging/global-business/cancel-message.php
<?php
require_once(__DIR__ . '/../vendor/autoload.php');

use Messente\Api\Api\OmnimessageApi;
use Messente\Api\Configuration;

$config = Configuration::getDefaultConfiguration()
    ->setUsername('YOUR_MESSENTE_API_USERNAME')
    ->setPassword('YOUR_MESSENTE_API_PASSWORD');

$apiInstance = new OmnimessageApi(new GuzzleHttp\Client(), $config);

try {
    $apiInstance->cancelScheduledMessage('<omnimessage_id>');
    echo 'Scheduled omnimessage cancelled', PHP_EOL;
} catch (Exception $e) {
    echo 'Exception when cancelling an omnimessage: ', $e->getMessage(), PHP_EOL;
}
